app/views/declaration/summary/parties_section.scala.html

```diff
@@ -83,21 +83,34 @@ <h2 class="govuk-heading-m">Parties</h2>
     )
     @summaryList("declaration-parties-summary", None,
 
-        declarationData.parties.exporterDetails.map(export => eoriOrAddress(
+        (Seq.empty :+ declarationData.parties.declarantIsExporter.map(isExporter =>
```

Comment:
Comment as always :)

Not `Seq.empty :+ but Seq(element)`

Reply:
It doesn't work.  I've tried it several times.  When I originally did the new summary page, when you last made that comment and this time.

Reply:
I checked this. It's because  `declarationData.parties.declarantIsExporter` is an Option, not Sequence.

So in that case you need to wrap whole block into the Sequence., not just this element.

Reply:
I think each "part" of this produces a Seq[Option[SummaryListRow]] - driven mainly by the `EoriOrAddress` helper.